Core/AppRuntime/Source/AppRuntimeV8.cpp

```diff
@@ -87,4 +93,19 @@ namespace Babylon
         // delete isolate->GetArrayBufferAllocator();
         isolate->Dispose();
     }
+
+    bool AppRuntimeV8::EnableInspector(unsigned short port)
+    {
+        s_inspectorPort = port;
+#ifdef V8_INSPECTOR_INCLUDED
+        return true;
+#else
+        return false;
+#endif
+    }

```

Comment:
This contract feels odd to me. Users can always call this function but it may not work if the code is not available?

Reply:
Yeah, I wasn't sure if throwing an exception would be more appropriate here, since there isn't a clean way to tell from App.cpp if the static library is included or not, and since we might not have the inspector on Android...

Perhaps it would be best to test on Android and then just have the inspector code always included in the V8 build to avoid the extra logic at the cost of a small binary size bump...